backend/requirements.txt

```diff
@@ -5,29 +5,24 @@ click==7.1.2
 flake8==3.8.4
 Flask==1.1.2
 idna==2.10
+importlib-metadata==2.0.0
 itsdangerous==1.1.0
 Jinja2==2.11.2
 kaggle==1.5.9
+lxml==4.6.1
 MarkupSafe==1.1.1
 mccabe==0.6.1
-numpy==1.19.4
-pandas==1.1.4
 Pillow==8.0.1
-praw==7.1.0
-prawcore==1.5.0
 pycodestyle==2.6.0
 pyflakes==2.2.0
 python-dateutil==2.8.1
 python-slugify==4.0.1
-pytz==2020.4
 requests==2.25.0
 six==1.15.0
 slugify==0.0.1
 soupsieve==2.0.1
 text-unidecode==1.3
 tqdm==4.52.0
-update-checker==0.18.0
 urllib3==1.26.2
```

Comment:
We're using `requests` so what's this for?

Reply:
I don't know too but it comes with kaggle-api, [setup file](https://github.com/Kaggle/kaggle-api/blob/master/setup.py)

Reply:
OK then